### ISSUE TITLE
save dialog popup when close excel manually.

### ISSUE THREAD

**Issue by shsx477** [EPPlusSoftware/EPPlus#890]
sorry for not good at english.

```
using (var package = new ExcelPackage(this.FilePath))
{
    ExcelWorksheet worksheet = package.Workbook.Worksheets[0];
    ExcelWorksheet newWorksheet = package.Workbook.Worksheets.Add("Sheet2", worksheet);
    package.Save();
}
```

reproduce
1. open excel file with excel.
2. close.
3. save dialog popup up

![sample](https://github.com/EPPlusSoftware/EPPlus/assets/10467151/0a79297c-255f-4f9a-adcc-62a6c54f6e61)

**Comment by swmal** [comment]
Hello,

Can you upload your original workbook here so we can try to replicate the issue?

**Comment by shsx477** [comment]
It also happends when i create an excel file and test it.
But i attach the file i used.

[test1.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/11630960/test1.xlsx)

**Comment by swmal** [comment]
Hello again,

we have identified the issue causing this and the fix will be included in the next version (to be released today or tomorrow).

**Comment by swmal** [comment]
This should be fixed in EPPlus 6.2.6.
